airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/GET_SELLER_FEEDBACK_DATA.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "Seller Feedback Data Reports",
+  "description": "Seller Feedback Data Reports",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "Date": {
+      "type": ["null", "string"],
+      "format": "date"
+    },
+    "Rating": {
+      "type": ["null", "number"]
+    },
+    "Comments": {
+      "type": ["null", "string"]
+    },
+    "Response": {
+      "type": ["null", "string"]
+    },
+    "Order ID": {
+      "type": ["null", "string"]
+    },
+    "Rater Email": {
+      "type": ["null", "string"]
```

Comment:
Is it the name of this fields correct with the raw data/response from the API?

Reply:
Yes, fields are named exactly like this